sdk/src/test/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/MerchantOrderResultTest.java

```diff
@@ -1,22 +1,40 @@
 package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
 
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
 import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.Currency;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.PaymentBrand;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionStatus;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionType;
+
 import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
 import org.json.JSONObject;
 import org.junit.Test;
 
+import java.util.Arrays;
+
 import static org.junit.Assert.assertEquals;
 
 public class MerchantOrderResultTest {
 
     @Test
     public void getSignatureData_Should_ReturnCorrectSignatureData() {
```

Comment:
Would it be good to also include a test case when the MerchantOrderResult does not contain any transactions? (this can happen for example when the consumer immediately cancelled the order).